packages/RichTextEditing-Core.package/RichTextEditorModel.class/instance/positionInTextAtIndex..st

```diff
@@ -0,0 +1,12 @@
+document information
+positionInTextAtIndex: anIndex
+
+	| lineCounter currentLine positionInLine |
+
+	lineCounter := 0.
+	currentLine := self displayedLines 
+		detect: [: each | lineCounter := lineCounter + 1. each first <= (anIndex + 1) and: [each last >= (anIndex + 1)]] 
```

Comment:
line too long

Reply:
What would correct formatting look like here?

Reply:
perhaps new line before `and:`